package.json

```diff
@@ -37,11 +38,11 @@
     "reselect": "^3.0.1",
     "rxjs": "5.4.0",
     "ts-helpers": "^1.1.2",
-    "zone.js": "^0.8.11",
```

Comment:
Dependencies are listed alphabetically so that they are easy to find and organizable.